csi/src/node.rs

```diff
@@ -96,6 +96,32 @@ fn get_access_type(
     }
 }
 
+/// Detach the nexus device from the system, either at volume unstage,
+/// or after failed filesystem mount at volume stage.
+async fn detach(uuid: &Uuid, errheader: String) -> Result<(), Status> {
+    if let Some(device) = Device::lookup(uuid).await.map_err(|error| {
+        failure!(
+            Code::Internal,
+            "{} error locating device: {}",
+            &errheader,
+            error
+        )
```

Comment:
Should we include the uuid of the device in this error?

Reply:
That information is included in the `errheader` string, i.e. `msg.volume_id`.  Invocations of the `detach` function on lines 485 and 546.